.github/config/linux-web-tests.json

```diff
@@ -6,7 +6,6 @@
   "web_tests": true,
   "targets": [
     "blink_web_tests",
-    "content_shell",
```

Comment:
@hlwarriner why doesn't this fail? Does blink_web_tests build cobalt_shell instead?